apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -697,13 +700,36 @@ async function openLinksInParentFrame( calypsoPort ) {
 		'.interface-interface-skeleton__sidebar, .interface-interface-skeleton__secondary-sidebar'
 	);
 	for ( const sidebar of sidebars ) {
-		observeSidebarMutations( sidebar );
+		handleSidebarNodeAdded( sidebar );
 	}
 
 	// Add and remove the sidebar observers as the sidebar elements appear and disappear.
 	// They are always direct children of the body element.
-	const body = document.querySelector( '.interface-interface-skeleton__body' );
-	sidebarsObserver.observe( body, { childList: true } );
+	const sidebarBodyElem = document.querySelector( '.interface-interface-skeleton__body' );
+	sidebarsObserver.observe( sidebarBodyElem, { childList: true } );
+
+	const popoverSlotObserver = new window.MutationObserver( ( mutations ) => {
+		const isComponentsPopover = ( node ) => node.classList.contains( 'components-popover' );
+
+		const replaceWithManageReusableBlocksHref = ( anchorElem ) => {
+			anchorElem.ref = manageReusableBlocksUrl;
```

Comment:
On a second round of review I found the following typo (missing `h`):

```suggestion
			anchorElem.href = manageReusableBlocksUrl;
```

As a result, the link still points to the WP Admin page instead of the Calypso one (which I didn't realize before).

This is how the page should look like:
![Markup on 2022-04-05 at 13:21:22](https://user-images.githubusercontent.com/25105483/161743296-989be292-abc3-40a0-b84c-c79dd0068997.png)

instead of:
![Markup on 2022-04-05 at 13:21:35](https://user-images.githubusercontent.com/25105483/161743337-b529343f-d269-4fa0-a885-2d36ce7a54ec.png)